### PR TITLE
update FSAC, purge old mono-referencing options, update options texts

### PR DESCRIPTION
**Pull request by baronfel**
Polishing before release, wanted to run these by @Krzysztof-Cieslak before merge.

### CODE REVIEW

File: release/package.json [hunk 1]
```diff
@@ -1177,18 +1177,6 @@
 					"default": 4,
 					"description": "The deep level of directory hierarchy when searching for sln/projects"
 				},
-				"FSharp.monoPath": {
```
Comment (baronfel) [inline release/package.json:1180]: mono is no longer used, so I purged the option

File: release/package.json [hunk 2]
```diff
@@ -1177,18 +1177,6 @@
...
-					"type": "string",
-					"default": "mono",
-					"description": "The path to Mono executable",
-					"scope": "machine-overridable"
-				},
-				"FSharp.fsiFilePath": {
```
Comment (baronfel) [inline release/package.json:1186]: we don't really support .net fx scripting anymore, do we?

File: release/package.json [hunk 3]
```diff
@@ -1273,38 +1261,32 @@
...
 					"default": [],
-					"description": "Allows to pass extra parameters to FSI process"
+					"description": "An array of additional command line parameters to pass to FSI when it is started. See [the documentation](https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/fsharp-interactive-options) for an exhaustive list."
 				},
 				"FSharp.saveOnSendLastSelection": {
 					"type": "boolean",
 					"default": false,
-					"description": "Save Current file before send LastSelection to FSI"
-				},
-				"FSharp.msbuildLocation": {
```
Comment (baronfel) [inline release/package.json:1288]: we don't support .net fx msbuild anymore, the code in the msbuild module wasn't ever using this

File: release/package.json [hunk 4]
```diff
@@ -1203,62 +1191,62 @@
...
 					"type": "boolean",
 					"default": false,
-					"description": "Includes external (from unopen modules and namespaces) symbols in autocomplete"
+					"description": "Includes external (from unopened modules and namespaces) symbols in autocomplete"
 				},
 				"FSharp.linter": {
 					"type": "boolean",
 					"default": true,
-					"description": "Enables integration with FSharpLinter (additional warnings)"
+					"description": "Enables integration with [FSharpLint](https://fsprojects.github.io/FSharpLint/) for additional (user-defined) warnings"
```
Comment (Krzysztof-Cieslak) [inline release/package.json:1199]: Is this displayed nicely in VSCode's settings panel?

File: release/package.json [hunk 5]
```diff
@@ -1329,18 +1311,9 @@
 				},
 				"FSharp.lineLens.prefix": {
 					"type": "string",
-					"description": "The prefix displayed before the signature",
+					"description": "The prefix displayed before the signature in a LineLens",
 					"default": "  // "
 				},
-				"FSharp.tooltipMode": {
```
Comment (baronfel) [inline release/package.json:1335]: unused as far as I can see in the product

File: release/package.json [hunk 6]
```diff
@@ -1468,16 +1441,6 @@
 					"default": true,
 					"description": "Allow Ionide to prompt whenever internal data files aren't included in your project's .gitignore"
 				},
-				"FSharp.suggestSdkScripts": {
```
Comment (baronfel) [inline release/package.json:1471]: we're all in on .net sdk now

File: release/package.json [hunk 7]
```diff
@@ -1468,16 +1441,6 @@
...
-					"type": "boolean",
-					"default": true,
-					"description": "Allow Ionide to prompt to use SdkScripts"
-				},
-				"FSharp.useSdkScripts": {
```
Comment (baronfel) [inline release/package.json:1476]: we're all in on .net sdk now

File: src/Core/LanguageService.fs [hunk 8]
```diff
@@ -553,61 +553,11 @@ Consider:
...
-                let! fsiPath = getAnyCpuFsiPathFromCompilerLocation fsacPaths
-                return fsiPath
-        }
-
     let fsiSdk () =
         promise {
             return Environment.configFsiSdkFilePath ()
         }
 
-    let fsc () =
```
Comment (baronfel) [inline src/Core/LanguageService.fs:593]: this wasn't used anywhere

File: src/Core/LanguageService.fs [hunk 9]
```diff
@@ -553,61 +553,11 @@ Consider:
...
-        promise {
-            match Environment.configFSCPath () with
-            | Some path -> return Some path
-            | None ->
-                let! fsacPaths = fsacConfig ()
-                return fsacPaths.Fsc
-        }
-
-    let msbuild () =
```
Comment (baronfel) [inline src/Core/LanguageService.fs:602]: we don't support .net fx msbuild, so this can go in its entirety

File: release/package.json (hunk 4 above)
Comment (baronfel) [reply release/package.json:1199]: checking these as we speak

Comment (baronfel) [reply release/package.json:1199]: poop

image

File: release/package.json (hunk 2 above)
Comment (Krzysztof-Cieslak) [reply release/package.json:1186]: I think it's still possible to do full framework scripting. As far as I remember it was related to the fact that there are breaking changes between both FSI implementations so users may want to use full framework FSI

File: release/package.json (hunk 4 above)
Comment (baronfel) [reply release/package.json:1199]: there we go 

image

File: release/package.json (hunk 2 above)
Comment (baronfel) [reply release/package.json:1186]: added fsiFilePath, useSdkScript, and suggestSdkScripts back, as well as the related code in Environment.fs, LanguageService.fs and Fsi.fs

File: release/package.json (hunk 6 above)
Comment (baronfel) [reply release/package.json]: reverted this so that we have all the fsi selection stuff back

File: release/package.json (hunk 7 above)
Comment (baronfel) [reply release/package.json:1476]: reverted this as well, all the old fsi switch stuff is back

**Pull request merged**